reana_commons/config.py

```diff
@@ -336,6 +336,11 @@ def kubernetes_node_label_to_dict(node_label):
 SHARED_VOLUME_PATH = os.getenv("SHARED_VOLUME_PATH", "/var/reana")
 """Default shared volume path."""
 
+DEFAULT_WORKSPACE_PATH = os.getenv("DEFAULT_WORKSPACE_PATH", "/var/reana")
+"""Default workspace path defined by the admin."""
+
+ADMIN_ALLOWED_WORKSPACES = os.getenv("ADMIN_ALLOWED_WORKSPACES", None)
```

Comment:
Minor: docstring is missing.

Perhaps we could split the string and append `SHARED_VOLUME_PATH` here to have the final list. In that case code will be simplified `reana_commons/workspaces.py` and this operation will be done only once

Reply:
I agree, good change, as I wont have to make that operation in the validation or showing the workspaces